opentelemetry-etw-logs/src/logs/exporter/common.rs

```diff
@@ -22,6 +23,7 @@ pub fn add_attribute_to_event(event: &mut tld::EventBuilder, key: &Key, value: &
         AnyValue::Bytes(b) => {
             event.add_binaryc(key.as_str(), b.as_slice(), tld::OutType::Default, 0);
         }
+        #[cfg(feature = "serde_json")]
```

Comment:
@psandana @cijothomas I reviewed the current JSON serialization logic and noticed that we don't restrict the recursion depth when serializing complex value types like ListAny and Map. This could result in performance issues and potentially cause a StackOverflow if a user-provided log with a deeply nested structure is processed by the exporter.

I think we should our custom serialization logic to enforce a maximum depth limit. I don't think `serde` offers any such configuration for serialization.

Reply:
We'd be having same issue in OTLP Exporters too, so need to address it there too.

Reply:
One way to fix would be #238